src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -8363,33 +8350,12 @@ static R2 F4(ref int i)
-                    // (16,16): error CS8352: Cannot use variable 'y1' in this context because it may expose referenced variables outside of their declaration scope
-                    //         return y1; // 1
-                    Diagnostic(ErrorCode.ERR_EscapeVariable, "y1").WithArguments("y1").WithLocation(16, 16),
-                    // (22,9): error CS8352: Cannot use variable 'r' in this context because it may expose referenced variables outside of their declaration scope
-                    //         r.Deconstruct(out var x2, out var y2);
-                    Diagnostic(ErrorCode.ERR_EscapeVariable, "r").WithArguments("r").WithLocation(22, 9),
-                    // (22,9): error CS8350: This combination of arguments to 'R1.Deconstruct(out int, out R2)' is disallowed because it may expose variables referenced by parameter 'this' outside of their declaration scope
-                    //         r.Deconstruct(out var x2, out var y2);
-                    Diagnostic(ErrorCode.ERR_CallArgMixing, "r.Deconstruct(out var x2, out var y2)").WithArguments("R1.Deconstruct(out int, out R2)", "this").WithLocation(22, 9),
-                    // (28,9): error CS8352: Cannot use variable 'var (x3, y3) = r' in this context because it may expose referenced variables outside of their declaration scope
-                    //         var (x3, y3) = r;
-                    Diagnostic(ErrorCode.ERR_EscapeVariable, "var (x3, y3) = r").WithArguments("var (x3, y3) = r").WithLocation(28, 9),
-                    // (28,24): error CS8350: This combination of arguments to 'R1.Deconstruct(out int, out R2)' is disallowed because it may expose variables referenced by parameter 'this' outside of their declaration scope
-                    //         var (x3, y3) = r;
-                    Diagnostic(ErrorCode.ERR_CallArgMixing, "r").WithArguments("R1.Deconstruct(out int, out R2)", "this").WithLocation(28, 24),
-                    // (34,9): error CS8352: Cannot use variable 'r' in this context because it may expose referenced variables outside of their declaration scope
-                    //         r.Deconstruct(out var x4, out var y4);
-                    Diagnostic(ErrorCode.ERR_EscapeVariable, "r").WithArguments("r").WithLocation(34, 9),
-                    // (34,9): error CS8350: This combination of arguments to 'R1.Deconstruct(out int, out R2)' is disallowed because it may expose variables referenced by parameter 'this' outside of their declaration scope
-                    //         r.Deconstruct(out var x4, out var y4);
-                    Diagnostic(ErrorCode.ERR_CallArgMixing, "r.Deconstruct(out var x4, out var y4)").WithArguments("R1.Deconstruct(out int, out R2)", "this").WithLocation(34, 9));
+                // (16,16): error CS8352: Cannot use variable 'y1' in this context because it may expose referenced variables outside of their declaration scope
```

Comment:
nit: consider indenting this to match the previous test baseline

Reply:
I'll update the indenting in a follow-up PR, unless other changes are needed in this PR.